router/network.go

```diff
@@ -243,6 +253,36 @@ func (network *netHandle) SendPost(ctx context.Context, structData integrations.
 	}
 }
 
+func (n *netHandle) doResponseLogging(resp responseLogDetails, destInfo types.DestinationInfo) {
+	isEnabled := config.GetReloadableBoolVar(false, "Router.Network.ResponseLogger.enabled").Load()
+	if !isEnabled {
+		return
+	}
+
+	destID := config.GetReloadableStringVar("", "Router.Network.ResponseLogger.destinationID").Load()
+	destType := config.GetReloadableStringVar("", "Router.Network.ResponseLogger.destType").Load()
+	workspaceID := config.GetReloadableStringVar("", "Router.Network.ResponseLogger.workspaceID").Load()
+
+	headerBytes, err := json.Marshal(resp.headers)
+	if err != nil {
+		n.logger.Debugn(fmt.Sprintf("header marshal failure: %s", err.Error()),
+			obskit.DestinationID(destInfo.ID),
+			obskit.DestinationType(destInfo.DefinitionName),
+			obskit.WorkspaceID(destInfo.WorkspaceID),
+		)
```

Comment:
```suggestion
		n.logger.Debugn("header marshal failure",
		    obskit.Error(err),
			obskit.DestinationID(destInfo.ID),
			obskit.DestinationType(destInfo.DefinitionName),
			obskit.WorkspaceID(destInfo.WorkspaceID),
		)
```

also, is the Debug the appropriate level here? I would go with a Warn

Reply:
Makes sense. This is done.